src/loopfilter.rs

```diff
@@ -384,3 +395,726 @@ extern "C" {
         bitdepth_max: c_int,
     );
 }
+
+// TODO(perl) Temporarily pub until mod is deduplicated
+#[inline(never)]
+pub(crate) unsafe fn loop_filter<BD: BitDepth>(
```

Comment:
Still more of these 🤣

Reply:
derp, sorry, I think I did them all now... :grimacing: